### PR TITLE
Default project

### LINKED ISSUE #594: js scripts as default loaded files
I think we can also include some relevant js scripts in our example files which are loaded by default.

### LINKED ISSUE #608: Improve Remix project template
For the moment a basic project includes example files and test files.
We should add to it JavaScript File (script file) for deploying / calling contract and a readme which explain how to use it.

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #594  
fixes #608

### CODE REVIEW

File: apps/remix-ide-e2e/src/tests/editor.test.ts [hunk 1]
```diff
@@ -97,9 +100,11 @@ module.exports = {
     .click('li[key="browser/removeSourcehighlightScript.js"]')
     .pause(2000)
     .executeScript('remix.exeCurrent()')
-    .waitForElementVisible('li[key="browser/3_Ballot.sol"]')
-    .click('li[key="browser/3_Ballot.sol"]')
-    .pause(2000)
+    .waitForElementVisible('li[key="browser/contracts"]')
+    .click('li[key="browser/contracts"]') // files don't appear, so we click twice to get the files
```
Comment (yann300) [inline apps/remix-ide-e2e/src/tests/editor.test.ts:104]: That sounds a bit weird, is this the only case this happens?

File: apps/remix-ide/src/app/editor/examples.js [hunk 2]
```diff
@@ -248,10 +248,92 @@ contract BallotTest {
...
-  owner: { name: '2_Owner.sol', content: owner },
-  ballot: { name: '3_Ballot.sol', content: ballot },
-  ballot_test: { name: 'tests/4_Ballot_test.sol', content: ballotTest }
+  storage: { name: 'contracts/1_Storage.sol', content: storage },
+  owner: { name: 'contracts/2_Owner.sol', content: owner },
+  ballot: { name: 'contracts/3_Ballot.sol', content: ballot },
+  deployWithWeb3: { name: 'scripts/deployWithWeb3.js', content: deployWithWeb3 },
+  deployWithEthers: { name: 'scripts/deployWithEthers.js', content: deployWithEthers },
+  ballot_test: { name: 'tests/4_Ballot_test.sol', content: ballotTest },
+  readme: { name: 'README', content: readme }
```
Comment (yann300) [inline apps/remix-ide/src/app/editor/examples.js:338]: there should probably be an extension. I'd `md` is fine (even if we don't have a md viewer), or `txt` could work too.

Comment (yann300) [inline apps/remix-ide/src/app/editor/examples.js:338]: in my test branch, I had `call.js` and `transact.js` , would it make sense to put it here too?

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/editor/examples.js:338]: yes definitely. May be we can have a separate PR for that.

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/editor/examples.js:338]: As we are putting simple text, I will go with `.txt`

File: apps/remix-ide-e2e/src/tests/editor.test.ts (hunk 1 above)
Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/editor.test.ts:104]: Yes. I will open an issue for this.

Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/editor.test.ts:104]: Here it is: https://github.com/ethereum/remix-project/issues/670

Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/editor.test.ts:104]: this also happens here: https://github.com/ethereum/remix-project/pull/659/files#diff-be796db79c0c7616c194328f2833fc50a1cfb57fe105aaa4a390f89e30aa1513R46

File: apps/remix-ide/src/app/editor/examples.js (hunk 2 above)
Comment (Aniket-Engg) [reply apps/remix-ide/src/app/editor/examples.js]: this is done

**Comment by yann300** [comment]
in each script, perhaps we should write something like:
`right click on the file name and hit "run" to execute`

**Comment by yann300** [comment]
the indentation of `deploy_web3.js` and `deploy_ethers.js` is different

**Comment by Aniket-Engg** [comment]
> in each script, perhaps we should write something like:
> `right click on the file name and hit "run" to execute`

done

**Comment by Aniket-Engg** [comment]
> the indentation of `deploy_web3.js` and `deploy_ethers.js` is different

improved

**Review by yann300 — APPROVED** [review]

**Pull request merged**